Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER hello@codeclimate.com
 
 RUN adduser -u 9000 -D app
-USER app
 
-CMD ["/usr/src/app/codeclimate-gofmt"]
+WORKDIR /code
+
+COPY build/ /usr/src/app
+COPY build/codeclimate-gofmt /usr/src/app/build/codeclimate-gofmt
```

Comment:
My comment got lost in out-of-date-diff land, so re-adding:

If you're already copying all of `build/` above, that means `/usr/src/app/codeclimate-gofmt` will exist. Why not use that instead of making a redundant layer to put it at a different location?